python/ray/autoscaler/kubernetes/operator_configs/helm/ray/templates/raycluster.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if not .Values.noRayCluster }}
+apiVersion: cluster.ray.io/v1
+kind: RayCluster
+metadata:
+  name: {{ .Release.Name }}
+spec:
+  # The maximum number of workers nodes to launch in addition to the head node.
+  maxWorkers: {{ include "ray.clusterMaxWorkers" . }}
+  # The autoscaler will scale up the cluster faster with higher upscaling speed.
+  # E.g., if the task requires adding more nodes then autoscaler will gradually
+  # scale up the cluster in chunks of upscaling_speed*currently_running_nodes.
+  # This number should be > 0.
+  upscalingSpeed: 1.0
+  # If a node is idle for this many minutes, it will be removed.
+  idleTimeoutMinutes: 5
+  # Specify the pod type for the ray head node (as configured below).
+  headPodType: {{ .Values.headPodType }}
+  # Specify the allowed pod types for this ray cluster and the resources they provide.
+  podTypes:
+    {{- range .Values.podTypes }}
+    - name: {{ .name }}
+      minWorkers: {{ .minWorkers }}
+      maxWorkers: {{ .maxWorkers }}
+      podConfig:
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          generateName: {{ .name }}-
+        spec:
+          restartPolicy: Never
+          # This volume allocates shared memory for Ray to use for its plasma
+          # object store. If you do not provide this, Ray will fall back to
+          # /tmp which cause slowdowns if is not a shared memory volume.
+          volumes:
+          - name: dshm
+            emptyDir:
+              medium: Memory
+          containers:
+          - name: ray-node
+            imagePullPolicy: Always
+            image: {{ $.Values.image }}
+            # Do not change this command - it keeps the pod alive until it is
+            # explicitly killed.
+            command: ["/bin/bash", "-c", "--"]
+            args: ['trap : TERM INT; sleep infinity & wait;']
+            ports:
+            - containerPort: 6379  # Redis port
+            - containerPort: 10001  # Used by Ray Client
+            - containerPort: 8265  # Used by Ray Dashboard
```

Comment:
May be useful to include the Ray Serve port on 8000 as well.

Reply:
From what I understand, a container's `ports` list functions pretty much as documentation https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports 
That said -- it's definitely a good idea to document the serve port here!